warehouse/filters.py

```diff
@@ -136,3 +137,8 @@ def format_classifiers(classifiers):
     structured = collections.OrderedDict(sorted(structured.items()))
 
     return structured
+
+
+def filter_valid_uris(items):
+    """ Filter out a list so that it only contains valid urls """
```

Comment:
Ok, sure. I always start / end mine with a single space because I think it makes the code appear less cluttered (and both are PEP8 acceptable), but if this team omits it that's fine by me.